### ISSUE TITLE
Does not close the file when muxing fails

### ISSUE THREAD

**Issue by NoxMortem** [facebook/360-Capture-SDK#28]
Which leaves both files in use and they can not be deleted.

**Comment by grancia** [comment]
Yes, files can't be deleted programmatically until the handle is closed.

**Comment by NoxMortem** [comment]
Which is why it might be a good idea to close the handle when the muxing fails?
